src/pages/Archive/Archive.jsx

```diff
@@ -0,0 +1,87 @@
+import { useNote } from "../../Context/noteContext";
+import { useAuth } from "../../Context/authContext";
+import axios from "axios";
+import { Sidebar } from "../../components/Sidebar/Sidebar";
+import {MdiRestore, MdiTrashCanOutline} from "../../assets/Icon/Icon";
+
+export function Archive() {
+    const {noteState, noteDispatch} = useNote();
+    const {token} = useAuth();
+
+    const archiveDataRestore = async(id) => {
+        try{
+            const res = await axios.post(`/api/archives/restore/${id}`,{}, {
+                headers: {
```

Comment:
Instead of manually  passing authorization token header in each request, try using it in axios interceptors. This will remove code repeatation.